examples/image-to-text/run_pipeline.py

```diff
@@ -67,12 +69,23 @@ def main():
         action="store_true",
         help="Whether to perform generation in bf16 precision.",
     )
+    parser.add_argument(
+        "--output_dir",
+        default=None,
+        type=str,
+        help="Output directory to store results in.",
+    )
     parser.add_argument("--batch_size", type=int, default=1, help="Input batch size.")
     parser.add_argument("--warmup", type=int, default=3, help="Number of warmup iterations for benchmarking.")
     parser.add_argument("--n_iterations", type=int, default=5, help="Number of inference iterations for benchmarking.")
     args = parser.parse_args()
 
     adapt_transformers_to_gaudi()
+    if args.image_path is None and "llava" in args.model_name_or_path:
```

Comment:
If we want to do this for all Llava models (i.e. also Llava models that don't have "llava" in their name), we should do:
```
from transformers import AutoConfig
model_type = AutoConfig.from_pretrained(args.model_name_or_path).model_type
if args.image_path is None and model_type == "llava":
```

Reply:
fixed